tests/e2e/taqueria-plugin-taquito-flextesa.spec.ts

```diff
@@ -0,0 +1,138 @@
+import fsPromises from "fs/promises"
+import { generateTestProject, getContainerName } from "./utils/utils";
+import { exec as exec1 } from "child_process";
+import utils from 'util'
+const exec = utils.promisify(exec1)
+
+const taqueriaProjectPath = 'e2e/auto-test-taquito-flextesa-plugin';
+const contractRegex = new RegExp(/(KT1)+\w{33}?/);
+let environment: string;
+let dockerName: string = "local"
+
+describe("E2E Testing for taqueria taquito plugin",  () => {
+
+    beforeAll(async () => {
+        await generateTestProject(taqueriaProjectPath, ["taquito", "flextesa"]);
+        await exec(`cp e2e/data/config-taquito-flextesa-local-sandbox-test-environment.json ${taqueriaProjectPath}/.taq/config.json`);
+        await exec(`cp e2e/data/hello-tacos.tz ${taqueriaProjectPath}/artifacts/`)
+        await exec(`cp e2e/data/increment.tz ${taqueriaProjectPath}/artifacts/`)
+        await exec(`taq stop sandbox ${dockerName}`, {cwd: `./${taqueriaProjectPath}`})
+        await exec(`taq start sandbox ${dockerName}`, {cwd: `./${taqueriaProjectPath}`})
+    })
+
+    // TODO: Consider in future to use keygen service to update account balance programmatically
+    // https://github.com/ecadlabs/taqueria/issues/378
+    test('Verify that taqueria taquito plugin can deploy one contract using deploy command', async () => {
```

Comment:
If the compiled contracts are getting moved to the beforeAll section then this test isn't any different from the last test and can be removed